tools/offline_log_viewer/controller.py

```diff
@@ -659,22 +659,28 @@ def decode_user_and_credential(rdr: Reader):
         })
 
 
+def decode_bootstrap_cluster_cmd_data(rdr: Reader, version):
+    decoded = {
+        'cluster_uuid': rdr.read_uuid(),
+        'bootstrap_user_cred': rdr.read_optional(decode_user_and_credential),
+        'nodes_by_uuid': rdr.read_serde_map(Reader.read_uuid,
+                                            Reader.read_int32)
+    }
+    if version >= 1:
+        decoded |= {'founding_version': rdr.read_int64()}
```

Comment:
Maybe we should teach Python how to Serde at the byte level and then we wouldn't have to do stuff like this.

Reply:
what do you mean?